l10n_it_ricevute_bancarie/views/wizard_due_date_settlement.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="riba_due_date_settlement" model="ir.ui.view">
+        <field name="name">riba.due.date.settlement.wizard</field>
+        <field name="model">riba.due.date.settlement</field>
+        <field name="arch" type="xml">
+            <form>
+                <separator
+                    string="This wizard will set 'Paid' all C/O details selected by due date"
+                    colspan="4"
+                />
+                <group>
+                    <field name="due_date" />
+                </group>
+                <footer>
+                    <button
+                        string="Confirm"
+                        name="due_date_settlement_confirm"
+                        type="object"
+                        class="oe_highlight"
+                    />
+                    <button string="Cancel" special="cancel" class="oe_link" />
+                </footer>
+            </form>
+        </field>
+    </record>
+
+    <record id="action_wizard_riba_due_date_settlement" model="ir.actions.server">
```

Comment:
Non è necessaria un'azione server per lanciare un wizard.
Come mai questa scelta?

Reply:
senza azone serve il wizard non appare in "azione"

Reply:
@eLBati che suggeriresti?

Reply:
A me sembra sia comodo fatto così, quindi non lo cambierei.

Reply:
@eLBati buona per il merge? 🙏

Reply:
OK così